sdk/search/Azure.Search.Documents/src/Generated/Models/AnalyzerName.cs

```diff
@@ -312,7 +312,7 @@ public AnalyzerName(string value)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override bool Equals(object obj) => obj is AnalyzerName other && Equals(other);
         /// <inheritdoc />
-        public bool Equals(AnalyzerName other) => string.Equals(_value, other._value, StringComparison.Ordinal);
+        public bool Equals(AnalyzerName other) => string.Equals(_value, other._value, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
Were the extensible enum struct guidelines updated? This is a change from them. It was intentional, but open for change.

Reply:
Guidelines don't define any comparison. There is a sample that uses Ordinal but it's not explicitly mandated. https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-enums

Reply:
I know - I wrote that after discussing the finer points with the team. :) We should be consistent across implementations. If we're going to accept case-insensitive parsing, we should do so at least by default across implementations.

Reply:
@heaths @pakrym The service decides whether these things are case-insensitive or not, not the client. I don't think it's possible to have a generated implementation that works for the general case.